include/depends.mk

```diff
@@ -11,7 +11,7 @@
 
 DEP_FINDPARAMS := -x "*/.svn*" -x ".*" -x "*:*" -x "*\!*" -x "* *" -x "*\\\#*" -x "*/.*_check" -x "*/.*.swp" -x "*/.pkgdir*"
 
-find_md5=find $(wildcard $(1)) -type f $(patsubst -x,-and -not -path,$(DEP_FINDPARAMS) $(2)) -printf "%p%T@\n" | sort | mkhash md5
```

Comment:
It would be probably better to create `MKHASH=$(STAGING_DIR_HOST)/bin/mkhash` variable probably in `rules.mk` and use that `$(MKHASH) md5` instead of this copy&pasta. While at it, do `git grep mkhash` and make the usage of that consistent.